apps/xmtp.chat/src/components/Messages/WalletSendCalls.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button } from "@mantine/core";
+import type { Conversation } from "@xmtp/browser-sdk";
+import {
+  ContentTypeTransactionReference,
+  type TransactionReference,
+} from "@xmtp/content-type-transaction-reference";
+import type { WalletSendCallsParams } from "@xmtp/content-type-wallet-send-calls";
+import { useSendTransaction } from "wagmi";
+
+export type WalletSendCallsProps = {
+  content: WalletSendCallsParams;
+  sendMessage: Conversation["send"];
+};
+
+export const WalletSendCallsUI: React.FC<WalletSendCallsProps> = ({
+  content,
+  sendMessage,
+}) => {
+  const { sendTransactionAsync } = useSendTransaction();
+
+  function handleSubmit() {
+    void (async () => {
+      for (const call of content.calls) {
+        const wagmiTxData = {
+          ...call,
+          value: BigInt(parseInt(call.value || "0x0", 16)),
+          chainId: parseInt(content.chainId, 16),
+          gas: call.gas ? BigInt(parseInt(call.gas, 16)) : undefined,
+        };
+        const txHash = await sendTransactionAsync(wagmiTxData, {
+          onError(error) {
+            console.error(error);
+          },
+        });
+        const transactionReference: TransactionReference = {
+          networkId: content.chainId,
+          reference: txHash,
+        };
+        await sendMessage(
+          transactionReference,
+          ContentTypeTransactionReference,
+        );
+      }
+    })();
+  }
+
+  return (
+    <div>
```

Comment:
please use Mantine components for rendering content.

`div` => `Box`
`br` => `Divider`
`p` => `Text`

this ensure the theme is applied consistently across the app